patterns/1-initial/operational-model.md

```diff
@@ -0,0 +1,68 @@
+## Title
+
+InnerSource Operational Model
+
+## Patlet
+
+InnerSource is key across all of the software production chain. This includes from idea to delivery to customer. This pattern defines the software development life cycle from an InnerSource perspective.
+
+## Problem
+
+InnerSource is typically focused on letting developers work together. There are several patterns focused on specific roles as the Trusted Committer or from a more organization level as the Review Committee. However, it is still not clear how all of this works together.
+
+This pattern aims at providing an overview to management and Chief level of how the software life cycle may be in a more InnerSource way with the goal of having a place where InnerSource principles may apply to the several people involved: from Product Owners to Contributors.
+
+This may be seen as well as the glue for other patterns willing to make sense from a development lifecycle perspective.
+
+## Story (optional)
+
+## Context
+
+The content here is often tied to the applicability of the pattern for other readers: "Do I have this same particular situation?"
+
+The problem initially exists at the management level where the several business units are defining their needs, tuning the product, and suggesting next steps. A general overview of this process and key moments in the chain where they can influence (e.g., POs discussions) are part of this.
+
+## Forces
+
+As the software production chain involves roles with different goals, this way of working has to be part of everyone involved. Business related people as Product Owners may be reluctant to move into a more collaborative and transparent way of producing software. And this may happen with others as well.
```

Comment:
_Maybe_:  As the software production chain involves roles with different goals, the InnerSource way of working has to be taken up by everyone involved in the software production chain. For example, business related people as Product Owners may be reluctant to move into a more collaborative and transparent way of producing software. And this reluctance may occur with other roles as well.

**Comment**
Whenever possible, forces should be stated in a way that indicates the trade-offs (which can hint at the levers by which one can react to a given force). So if Product Owners are reluctant to be more collaborative and transparent, what is motivating them to do so? (Is it a cultural tendency to secrecy to protect proprietary information to gain a strategic advantage? If so, then is it a matter of noting *which* information *needs* to be protected vs. which info, when shared, can result in compensating advantages?)

The forces in this pattern likely need to be further brainstormed/developed/analyzed.

Reply:
Suggestion added.

With respect to the comment, I agree with you. However, this pattern could be seen as the umbrella for the other discussions. As an example, we're discussing on POs. What if we define more in detail the different stages? Or does it make sense to have this discussion in new patterns? (e.g., InnerSourcing Product Owners).